examples/gocean/eg3/alg.f90

```diff
@@ -65,8 +65,8 @@ program simple
 
   ! Create our grid
   model_grid = grid_type(GO_ARAKAWA_C,                                 &
-                         (/GO_BC_PERIODIC,GO_BC_PERIODIC,GO_BC_NONE/), &
-                         GO_OFFSET_SW)
+                         (/GO_BC_EXTERNAL,GO_BC_EXTERNAL,GO_BC_NONE/), &
+                         GO_OFFSET_NE)
```

Comment:
I changed it because with "GO_OFFSET_SW" it fails with:  `cv_sw_init: IMPLEMENT non-periodic BCs!`
I guess this is the limitation of dl_esm_inf described here: https://github.com/stfc/dl_esm_inf/issues/46

I now updated the kernels 'index_offset' also to GO_OFFSET_NE but I don't know if this is good enough or it is not as simple as just changing the kernel label.

Reply:
Unfortunately it's not quite as simple as that. I can't remember the details but it will require a shift in the array indices, maybe something like `i - 1` -> `i` for accesses to fields on U pts, `j - 1` -> `j` on V pts and both on F pts. See the picture in the https://psyclone.readthedocs.io/en/stable/gocean1p0.html#grid section. Since this is just an example the simplest solution may be to simply replace this kernel with one from `shallow` in PSycloneBench since that uses SW offset.

Reply:
Alternatively, you could try keeping GO_BC_PERIODIC although that might cause other problems?

Reply:
Reverted the changes and added a TODO to the relevant dl_esm_inf issue.